src/internal/m365/collection/site/metadata.go

```diff
@@ -97,3 +97,12 @@ func parseMetadataCollections(
 
 	return cdp[cat], true, nil
 }
+
+func pathFromPrevString(ps string) (path.Path, error) {
```

Comment:
We should also add this to the tech debt item. This func is also duplicated in 3 places. It can be moved to util or similar pkg.